Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -9646,9 +9642,9 @@ def __init__(self,
                  sort=None,
                  ):
         # searchAfter is available in searchIndicators from version 6.1.0
-        self._can_use_search_after = is_demisto_version_ge('6.1.0')
+        self._can_use_search_after = True
```

Comment:
@merit-maita it'll be a cleaner implementation if you remove this argument and its usage.
The same goes for `_can_use_filter_fields`.
These are feature flags - since they are no longer needed, the old implementation isn't needed either.